pyCFOFiSAX/_tree_iSAX.py

```diff
@@ -173,46 +173,46 @@ def nodes_visited_for_all_seq_ref(len_seq_list, distance,
 class TreeISAX:
     """
     La classe TreeISAX contenant
-     * la distribution iSAX des séquences à indexer
-     * et vers le premier nœud root
+     * ISAX distribution of sequences to index
+     * and towards the first root node
 
     .. WARNING::
-        Dans cette version, les data_ts sont obligatoires pour définir à l'avance les futurs breakpoints
+        In this version, the data_ts are mandatory to define in advance the future breakpoints
 
-    :param int size_word: le nombre de discrétisation SAX pour chaque séquence
-    :param int threshold: la capacité maximale des nœuds feuilles de l'arbre
-    :param numpy.ndarray data_ts: array de séquences à insérer
-    :param int base_cardinality: la plus petite cardinalité pour l'encodage iSAX
-    :param int max_card_alphabet: si self.boolean_card_max == True, cardinalite max pour l'encodage iSAX
+    :param int size_word: The number of Sax discretization for each sequence
+    :param int threshold: The maximum capacity of the nodes of the tree
+    :param numpy.ndarray data_ts: Sequence array to be inserted
+    :param int base_cardinality: The smallest cardinality for encoding iSAX
+    :param int max_card_alphabet: if self.boolean_card_max == True, Max cardinalite for encoding iSAX
```

Comment:
cardinalite (wrong word) -> (in french) cardinalité -> cardinality

Reply:
```suggestion
    :param int max_card_alphabet: if self.boolean_card_max == True, Max cardinality for encoding iSAX
```